first commit with overview page (#14827)
hashicorp/consul: website/content/docs/consul-vs-other/index.mdx

---
layout: docs
page_title: Why Choose Consul?
description: >-
  Consul is a service networking platform that centralizes service discovery, enables zero trust networking with service mesh, automates network infrastructure, and controls access to mesh services with the Consul API Gateway. Compare Consul with other software that provide similar capabilities with one or more of the core use cases.
---

# Why Choose Consul?

> HashiCorp Consul is a service networking platform that encompasses multiple capabilities to secure and simplify network service management. These capabilities include service mesh, service discovery, configuration management, and API gateway functionality. While competing products offer a few of these core capabilities, Consul is developed to address all four. The topics in this section provide a general overview of how Consul’s capabilities compare to some other tools on the market.

Reviewer: ```suggestion
HashiCorp Consul is a service networking platform that encompasses multiple capabilities to secure and simplify network service management. These capabilities include service mesh, service discovery, configuration management, and API gateway functionality. While competing products offer a few of these core capabilities, Consul is developed to address all four. The topics in this section provide a general overview of how Consul’s capabilities compare to some other tools on the market. Visit the following pages to read more about how:

- [Consul compares with other service meshes](/docs/consul-vs-other/service-mesh-compare)
- [Consul compares with other DNS tools](/docs/consul-vs-other/dns-tools-compare)
- [Consul compares with other configuration management tools](/docs/consul-vs-other/config-management-compare)
- [Consul compares with other API Gateways](/docs/consul-vs-other/api-gateway-compare)
```

---
layout: docs
page_title: Consul versus other service networking software
description: >-

>   Consul is service networking platform that addresses four core use cases relating to their network services: centralized service discovery, zero trust networking with service mesh, automating network infrastructure, and controlling access to mesh services with the Consul API Gateway. This doc will compare Consul with other software that provide similar capabilities with one or more of the core use cases.

Reviewer 2: ```suggestion
  Consul is a service networking platform that centralizes service discovery, enables zero trust networking with service mesh, automates network infrastructure, and controls access to mesh services with the Consul API Gateway. Compare Consul with other software that provide similar capabilities with one or more of the core use cases.
```
I'm not sure what the target SEO phrase is, but I might even cut ". . . is a service networking platform that . . ." to get it closer to the 180 word cut-off

---

> # Consul versus other software

Reviewer 2: ```suggestion
# Consul Compared to Other Software
```

> HashiCorp Consul is a service networking platform that encompasses multiple capabilities to secure and simplify management of network services. These capabilities are broken out into  four areas: service mesh, service discovery, configuration management, and API gateway functionality. While competing products offer a few of these core capabilities, Consul is developed to address all four. Below is a general overview of how Consul’s capabilities compare to some other tools on the market.

Reviewer 2: ```suggestion
HashiCorp Consul is a service networking platform that encompasses multiple capabilities to secure and simplify network service management. These capabilities include service mesh, service discovery, configuration management, and API gateway functionality. While competing products offer a few of these core capabilities, Consul is developed to address all four. The topics in this section provide a general overview of how Consul’s capabilities compare to some other tools on the market.
```

> HashiCorp Consul is a service networking platform that encompasses multiple capabilities to secure and simplify management of network services. These capabilities are broken out into  four areas: service mesh, service discovery, configuration management, and API gateway functionality. While competing products offer a few of these core capabilities, Consul is developed to address all four. Below is a general overview of how Consul’s capabilities compare to some other tools on the market.

Author: thanks Jeff, yeah that was due to changing from a single page to multiple pages, and I didn't catch the last sentence. Thanks!
